ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -784,8 +785,11 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
             height = [ORKSurveyAnswerCellForScale suggestedCellHeightForView:tableView];
         }
             break;
+        case ORKQuestionTypeVerticalScale:{
+            height = [ORKSurveyAnswerCellForVerticalScale suggestedCellHeightForView:tableView];
+        }
+            break;
```

Comment:
Yeah, I never felt good about adding a new `ORKQuestionTypeVerticalScale` value. I didn't realize `ORKQuestionStep` had this helper methods. I will improve it.